packages/@okta/vuepress-theme-default/components/Sidebar.vue

```diff
@@ -67,6 +92,24 @@
           })
         }
         return false
+      },
+
+      expandSubNav: function (event) {
+        var parent = event.target.parentElement;
+        var sections = parent.querySelector('.sections');
+        if(sections === null) {
+          return;
+        }
+
+        event.preventDefault();
+        
+        if(sections.style.display == 'block') {
+          parent.classList.remove('open');
+          sections.style.display = 'none';
+        } else {        
+          parent.classList.add("open");
+          sections.style.display = "block";
```

Comment:
not a complaint (I do it too), but the switch between single quote and double quote in this if/else is dizzying :)